src/frontends/pytorch/src/op/nms.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset8.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_nms(NodeContext& context) {
+    auto const_0 = context.mark_node(opset8::Constant::create(element::i64, Shape{}, {0}));
+    auto const_1 = context.mark_node(opset8::Constant::create(element::i64, Shape{}, {1}));
+    auto const_2 = context.mark_node(opset8::Constant::create(element::i64, Shape{1}, {2}));
+    // the shape that is required by PyTorch operator differs from the shape required in OpenVino
+    auto boxes_shape = context.mark_node(opset8::Constant::create(element::i64, Shape{3}, {1, -1, 4}));
+
+    auto boxes = context.mark_node(std::make_shared<opset8::Reshape>(context.get_input(0), boxes_shape, false));
```

Comment:
```suggestion
    auto boxes = context.mark_node(std::make_shared<opset8::Unsqueeze>(context.get_input(0), const_0));
```

Reply:
I also thought that using `Unsqueeze` operator is appropriate here, but when I use it, `NonMaxSuppression` operator returns an error. When I use `Reshape` operator and statically set first and last dimension, it is working correct.